src/views/Project/Details/General/StakeholdersModal/index.js

```diff
@@ -0,0 +1,307 @@
+import DangerButton from '#rsca/Button/DangerButton';
+import ModalBody from '#rscv/Modal/Body';
+import ModalFooter from '#rscv/Modal/Footer';
+import ModalHeader from '#rscv/Modal/Header';
+import ListView from '#rscv/List/ListView';
+
+import {
+    RequestCoordinator,
+    RequestClient,
+    methods,
+} from '#request';
+
+import _ts from '#ts';
+
+import SearchOrganization from './SearchOrganization';
+import OrganizationField from './OrganizationField';
+import styles from './styles.scss';
+
+
+const organizationKeySelector = d => d.id;
+const organizationLabelSelector = d => d.organizationDetails.title;
+const fieldKeySelector = d => d.faramElementName;
+
+const propTypes = {
+    closeModal: PropTypes.func.isRequired,
+    onChange: PropTypes.func.isRequired,
+    fields: PropTypes.array.isRequired, // eslint-disable-line react/forbid-prop-types
+    value: PropTypes.array.isRequired, // eslint-disable-line react/forbid-prop-types
+    requests: PropTypes.object.isRequired, // eslint-disable-line react/forbid-prop-types
+};
+
+const requestOptions = {
+    organizationsGetRequest: {
+        url: '/organizations/',
+        query: ({ params }) => ({
+            search: params && params.searchText,
+        }),
+        method: methods.GET,
+        onSuccess: ({ response: { results = [] }, params }) => {
+            if (params && params.handleSuccess) {
+                const { handleSuccess } = params;
+                handleSuccess(results);
+            }
+        },
+        options: {
+            delay: 300,
+        },
+    },
+};
+
+const transformOrganizationToOptions = (organizations = []) => (
+    organizations.map(({ id, organization, ...others }) => (
+        { ...others, id: organization, projectOrganizationId: id }
+    ))
+);
+
+const transformOrganizationToFaramValues = (organizations = []) => (
+    listToGroupList(organizations, o => o.organizationType, o => o.organization)
+);
+
+const transformFaramValuesToOrganization = (faramValues = {}, options = []) => {
+    let organizations = [];
+    Object.entries(faramValues).forEach(([key, values]) => {
+        const organizationList = values
+            .map(v => options.find(o => o.id === v))
+            .filter(isDefined)
+            .map(({
+                id,
+                projectOrganizationId,
+                organizationDetails,
+                organizationType,
+            }) => ({
+                id: organizationType === key ? projectOrganizationId : undefined,
+                organization: id,
+                organizationType: key,
+                organizationDetails,
+            }));
+
+        organizations = [...organizations, ...organizationList];
+    });
+    return organizations;
+};
+
+function uniqueAppend(list, elements, keySelector) {
+    const map = listToMap(list, keySelector);
+    elements.forEach((el) => {
+        if (!isDefined(map[keySelector(el)])) {
+            list.push(el);
+        }
+    });
+}
```

Comment:
We already have a unique function.why not just use unique([...oldArray, ...newArray], keySelector])